diagnostic.md

```diff
@@ -3,8 +3,10 @@
 Using `inspect element`, write four selectors found on the main page for this
 GitHub repo.
 
-<!-- your answer starts here -->
-
+.header-logged-in
+.site-footer-container
+.facebook
+.ajax-loader-bar
```

Comment:
It seems like you don't have the _entire_ selector on a few of these. `header-logged-in`, for example, is actually `header header-logged-in true`. Also, I think `facebook` is supposed to be `facebox`?